wcomponents-theme/build-resource.xml

```diff
@@ -28,10 +28,10 @@
 		</copy>
 
 		<copy todir="${target.rootdir}/resource" overwrite="true">
-			<fileset dir="${common.src.rootdir}/resource" includes="**" excludes="**.xml, **.rdf, **.html, **.svg, **.mustache, **.handlebars" />
```

Comment:
This will prevent the copy of the theme file `src/main/resource/wc.ui.timeoutWarn.handlebars` but that file has not been updated so requires this resource. The template itself is a simple template so could be a mustache template an could be renamed in the JavaScript.